cody.el

```diff
@@ -174,59 +187,63 @@ Returns nil if it cannot find it for any reason."
 The node version is only checked on Cody startup.
 You can call `cody-restart' to force it to re-check the version.")
 
-(defvar cody-node-path-override nil
-  "Hardwired path to the node.js binary to use for Cody.")
+(defvar cody--unit-testing-p nil
+  "Set to non-nil during unit testing.
+When testing, all calls to the agent are diverted..")
 
 (defun cody--agent-command ()
   "Command and arguments for running agent."
-  (list (or cody-node-path-override "node") cody--cody-agent))
+  (list (or cody-node-executable "node") cody--cody-agent))
 
 (defvar cody--connection nil "Global jsonrpc connection to Agent.")
 (defvar cody--message-in-progress nil "Chat message accumulator.")
-(defvar cody--access-token nil "LLM access token.")
+
+(defvar cody--sourcegraph-host "sourcegraph.com" "Sourcegraph host.")
+(defvar cody--access-token nil "LLM access token for `cody--sourcegraph-host'.")
```

Comment:
Realize you're just elaborating what was here, but while you're here... this isn't really an LLM access token per se, it is a "whole shooting box and dice, admit one" token. Like your remote list of repos and keyword search would all be via this token.